transformer_engine/jax/cpp_extensions.py

```diff
@@ -124,6 +124,15 @@ def _check_valid_batch_dims(bdims):
             f"but got {dim=}"
 
 
+# The value of activations are explicitedly defined to avoid order rearrangement in the future
+ActivationEnum = {
+    ('gelu',): 0,
+    ('gelu', 'linear'): 1,
+    ('silu',): 2,
+    ('silu', 'linear'): 3
+}
```

Comment:
I prefer to expose the C++ ActivationEnum to here through extensions.cpp, this can generate a hard dependency between python/c++'s enum to prevent the misalignments for the future development. You can refer `NVTE_QKV_Layout`.